osu.Game/Beatmaps/BeatmapDifficulty.cs

```diff
@@ -26,7 +26,7 @@ public float ApproachRate
             set => approachRate = value;
         }
 
-        public double SliderMultiplier { get; set; } = 1;
+        public double SliderMultiplier { get; set; } = 1; // AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA this needs to be renamed or removed or something.
```

Comment:
Stable internally seems to use `DifficultySliderMultiplier`, incidentally, which I like better than this. But that's for later.

Reply:
The hope would be to remove this completely, as it should only really be required as an editor "map default" if anything, once we're done.

Reply:
You mean just the prefix? I think it's about the same since this whole class is difficulty-grouped attributes of the beatmap.

Reply:
Yeah now that the other instances of "slider multiplier" have been culled it may be fine to leave this be as no longer ambiguous.